src/akeneo/api/catalogs/index.ts

```diff
@@ -7,15 +7,15 @@ import { paginateItemsBySearchAfter } from "..";
  * @param catalogId The catalog ID.
  * @returns The product identifiers.
```

Comment:
Super minor doc thingy - also check the function description above

```suggestion
 * @returns The product uuids.
```